File: Find unique element/Find_unique_element.cpp
class Solution {
  public:
    int findUnique(int k, vector<int>& arr) {
        unordered_map<int,int> mpp;
        for(auto i: arr) {
            mpp[i]++;
        }
        for(auto i: mpp) {
            if(i.second%k != 0) return i.first;
        }
        return -1;
    }
};

File: Find unique element/Find_unique_element.py
class Solution:
    def find_unique(self, k, arr):
        mpp = {}

        for i in arr:
            mpp[i] = mpp.get(i, 0) + 1

        for key, value in mpp.items():
            if value % k != 0:
                return key
        return -1
